src/k8s/pkg/component/component_test.go

```diff
@@ -266,10 +274,76 @@ func TestEnableMultipleComponents(t *testing.T) {
 		g.Expect(err).ShouldNot(HaveOccurred())
 	}
 
-	components, err := mockHelmClient.List()
-	g.Expect(err).ShouldNot(HaveOccurred())
-	g.Expect(components).To(ConsistOf(
-		Component{Name: "one", Status: true},
-		Component{Name: "two", Status: true},
-	), "Expected all components to be enabled")
+	// Reenable should not error
+	for name := range mockHelmClient.components {
```

Comment:
nit: you could put those steps into sub-tests:

```
t.Run("Reenable", func(...) {...})
```
That way, it is easier to see what went wrong.